test/functional/Valhalla/build.xml

```diff
@@ -54,6 +56,7 @@
 		<echo>===destdir:                      ${DEST}</echo>
 		<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" encoding="ISO-8859-1">
 			<src path="${src}"/>
+			<!-- <src path="${src_lw5}"/> -->
```

Comment:
Better to add a comment here on this line so that other people looking at this file knows why `<src path="${src_lw5}"/>` is commented out.